apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx

```diff
@@ -318,13 +318,14 @@ export const AttributeSettings: FunctionComponent<AttributeSelectionPropsInterfa
     const getExternalClaimsGroupedByScopes = () => {
         const tempClaims = [ ...externalClaims ];
         const updatedScopes = [];
-
+        const scopedClaims = [];
```

Comment:
Shall we type this?

Reply:
```suggestion
        const scopedClaims: ExtendedExternalClaimInterface[] = [];
```

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3537/commits/fe3791422914bf046739797bffccb53b7a9e248d